From 6592acda35f721ef2f06b423caeba33c20563585 Mon Sep 17 00:00:00 2001
From: Feng Shu <tumashu@163.com>
Date: Tue, 26 Dec 2023 11:21:17 +0800
Subject: [PATCH] Add g_debug for guix.

---
 liblightdm-gobject/session.c | 8 ++++++++
 liblightdm-gobject/system.c  | 1 +
 2 files changed, 9 insertions(+)

diff --git a/liblightdm-gobject/session.c b/liblightdm-gobject/session.c
index 79dcf466..9474d096 100644
--- a/liblightdm-gobject/session.c
+++ b/liblightdm-gobject/session.c
@@ -58,12 +58,14 @@ static GList *remote_sessions = NULL;
 static gint
 compare_session (gconstpointer a, gconstpointer b)
 {
+    g_debug ("YYY: Run compare_session function of liblightdm-gobject.");
     return strcmp (lightdm_session_get_name (LIGHTDM_SESSION (a)), lightdm_session_get_name (LIGHTDM_SESSION (b)));
 }
 
 static LightDMSession *
 load_session (GKeyFile *key_file, const gchar *key, const gchar *default_type)
 {
+    g_debug ("YYY: Run load_session function of liblightdm-gobject.");
     if (g_key_file_get_boolean (key_file, G_KEY_FILE_DESKTOP_GROUP, G_KEY_FILE_DESKTOP_KEY_NO_DISPLAY, NULL) ||
         g_key_file_get_boolean (key_file, G_KEY_FILE_DESKTOP_GROUP, G_KEY_FILE_DESKTOP_KEY_HIDDEN, NULL))
         return NULL;
@@ -203,6 +205,7 @@ update_sessions (void)
         remote_sessions_dir = value;
     }
 
+    g_debug ("Sessions_dir %s", sessions_dir);
     local_sessions = load_sessions (sessions_dir);
     remote_sessions = load_sessions (remote_sessions_dir);
 
@@ -219,6 +222,7 @@ update_sessions (void)
 GList *
 lightdm_get_sessions (void)
 {
+    g_debug ("YYY: Run lightdm_get_sessions function of liblightdm-gobject.");
     update_sessions ();
     return local_sessions;
 }
@@ -308,6 +312,7 @@ lightdm_session_get_comment (LightDMSession *session)
 static void
 lightdm_session_init (LightDMSession *session)
 {
+    g_debug ("YYY: Run lightdm_session_init function of liblightdm-gobject.");
 }
 
 static void
@@ -346,6 +351,7 @@ lightdm_session_get_property (GObject    *object,
 static void
 lightdm_session_finalize (GObject *object)
 {
+    g_debug ("YYY: Run lightdm_session_finalize function of liblightdm-gobject.");
     LightDMSession *self = LIGHTDM_SESSION (object);
     LightDMSessionPrivate *priv = lightdm_session_get_instance_private (self);
 
@@ -358,6 +364,8 @@ lightdm_session_finalize (GObject *object)
 static void
 lightdm_session_class_init (LightDMSessionClass *klass)
 {
+    g_debug ("YYY: Run lightdm_session_class_init function of liblightdm-gobject.");
+
     GObjectClass *object_class = G_OBJECT_CLASS (klass);
 
     object_class->set_property = lightdm_session_set_property;
diff --git a/liblightdm-gobject/system.c b/liblightdm-gobject/system.c
index 6b7dd19a..8b37ebc4 100644
--- a/liblightdm-gobject/system.c
+++ b/liblightdm-gobject/system.c
@@ -29,6 +29,7 @@
 const gchar *
 lightdm_get_hostname (void)
 {
+    g_debug ("YYY: Run lightdm_get_hostname function of liblightdm-gobject.");
     return g_get_host_name ();
 }
 
-- 
2.39.2

